src/main/java/com/parkhomovsky/bookstore/service/implementation/BookServiceImpl.java

```diff
@@ -0,0 +1,28 @@
+package com.parkhomovsky.bookstore.service.implementation;
+
+import com.parkhomovsky.bookstore.model.Book;
+import com.parkhomovsky.bookstore.repository.BookRepository;
+import com.parkhomovsky.bookstore.service.BookService;
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class BookServiceImpl implements BookService {
+    private final BookRepository bookRepository;
+
+    @Autowired
+    public BookServiceImpl(BookRepository bookRepository) {
+        this.bookRepository = bookRepository;
+    }
+
+    @Override
+    public Book save(Book book) {
+        return bookRepository.save(book);
+    }
+
+    @Override
+    public List<Book> findAll() {
```

Comment:
in service we use create, get naming for repository we use save and find naming

Reply:
in this case getAll()

Reply:
Can you rebuild your sentence? I don`t understand